scripts/packagelockHttps.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find . -type f -name \"package-lock.json\" | grep -v node_modules | xargs -I % sh -c 'echo \"Replacing http->https in %\"; sed -i \"\" -e \"s/http:\\/\\//https:\\/\\//g\" %';
```

Comment:
Is this a good time to bring up that it's impossible to use `sed -i ""` in a way that will run on both GNU and BSD sed?
https://stackoverflow.com/q/5694228
you could instead
```suggestion
find . -type f -name \"package-lock.json\" | grep -v node_modules | xargs -I % sh -c 'echo \"Replacing http->https in %\"; perl -pi -e \"s/http:\\/\\//https:\\/\\//g\" %';
```
(see https://stackoverflow.com/a/22247781)

Reply:
Also this script looks like it is overescaped - you probably need to remove one level of `\`

Reply:
This is out of scope for this issue imo, please make a new issue regarding this as this only runs on local machines so it should be fine as it is.

Reply:
Fair enough. Regardless, this script is overescaped and will not run correctly.

```suggestion
find . -type f -name "package-lock.json" | grep -v node_modules | xargs -I % sh -c 'echo "Replacing http->https in %"; sed -i "" -e "s/http:\/\//https:\/\//g" %';
```
(I've only written this in github - please also test that it is working locally)